testbench/scripts/get-tokens.sh

```diff
@@ -0,0 +1,4 @@
+#! /bin/bash
+
+source .env
```

Comment:
Can the python code be written in such a way that we don't have to source env everytime on a new terminal instance?